test/deleteReport.test.js

```diff
@@ -3,6 +3,7 @@ process.env.NODE_ENV = 'test';
 const supertest = require('supertest');
 const getPort = require('get-port');
 const {Op} = require('sequelize');
+const uuidv4 = require('uuid/v4')
```

Comment:
Why did you add this? It is never used?

Reply:
ah, that must have got missed. I added it when I was trying a different setup for mocking keycloak but I forgot to remove it again